addons/sourcemod/scripting/szf/config.sp

```diff
@@ -138,9 +138,24 @@ public ArrayList Config_LoadWeaponData()
 					int color[4];
 					kv.GetColor4("color", color);
 					
-					wep.color[0] = color[0];
-					wep.color[1] = color[1];
-					wep.color[2] = color[2];
+					wep.iColor[0] = color[0];
+					wep.iColor[1] = color[1];
+					wep.iColor[2] = color[2];
+					
+					float flOffsetOrigin[3];
+					float flOffsetAngles[3];
+					kv.GetVector("offset_origin", flOffsetOrigin);
+					kv.GetVector("offset_angles", flOffsetAngles);
```

Comment:
Any problem into using this rather than having new vector and through each 3 to set?
```kv.GetVector("offset_angles", wep.flOffsetAngles);```
Same thing to `wep.iColor` and `wep.flOffsetOrigin`

Reply:
Compiler throws an `array sizes do not match, or destination array is too small` error whenever i try to do it the easy way